.github/workflows/pytest.yml

```diff
@@ -169,22 +173,41 @@ jobs:
           workload_identity_provider: "projects/345950277072/locations/global/workloadIdentityPools/gh-actions-pool/providers/gh-actions-provider"
           service_account: "tox-pytest-github-action@catalyst-cooperative-pudl.iam.gserviceaccount.com"
 
-      - name: Run integration tests, trying to use GCS cache if possible
+      - name: Run ferc_to_sqlite
+        env:
+          COVERAGE_FILE: .coverage.ferc_to_sqlite
         run: |
-          pip install --no-deps --editable .
-          pudl_datastore --dataset epacems --partition year_quarter=2022q1
-          make pytest-integration
-
+          coverage run --concurrency=multiprocessing \
+            src/pudl/ferc_to_sqlite/cli.py --clobber ${{ env.ETL_COMMANDLINE_OPTIONS }} ${{ env.ETL_CONFIG }}
+      - name: Run pudl_etl
+        env:
+          COVERAGE_FILE: .coverage.pudl_etl
+        run: |
+          alembic upgrade head
+          coverage run --concurrency=multiprocessing \
+            src/pudl/etl/cli.py ${{ env.ETL_COMMANDLINE_OPTIONS }} ${{ env.ETL_CONFIG }}
+      - name: Run integration tests
+        env:
+          COVERAGE_FILE: .coverage.pytest
+        run: |
+          coverage run --concurrency=multiprocessing \
+           -m pytest -n auto --live-dbs test/integration
+      - name: Checksum coverage files
+        run: ls .coverage* | xargs md5sum | sort
+      - name: Generate coverage
+        run: |
+          coverage --version
+          coverage combine
```

Comment:
Why do we need to do `coverage combine` here? Is it because we've got a bunch of coverage files from various different processes?